lib/order.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'dishes'
+require_relative 'total'
+require_relative 'calculator'
+
+class Order < Calculator
+  include Dishes
+
+  attr_accessor :verify_order, :user_total, :order
+
+  def choose_dishes(*order, user_total)
```

Comment:
Does the user total need to be checked when they place their order? Could this code be simplified by doing that in a separate step?